Frontend/.env

```diff
@@ -1 +1 @@
-VITE_BACKEND_ADDRESS= http://localhost:3001
+VITE_BACKEND_ADDRESS= http://localhost:3002
```

Comment:
just curious why you had to change it from 3001 to 3002?

Reply:
For some reason, it said that 3001 was already in use when i tried to run my backend